src/Job/DeleteStaticCacheJob.php

```diff
@@ -34,7 +34,6 @@ public function process()
             }
             $meta = Publisher::singleton()->purgeURL($url);
             if (!empty($meta['success'])) {
-                $this->jobData->ProcessedURLs[$url] = $url;
```

Comment:
Same here, why don't we need this?

Reply:
@andrewandante This is a bit controversial but below are my points:
1. This is a single job for a single page, it is enough to know the that processed url is the page itself
2. There isn't any unit tests that requires this, hence removing this code still makes the test passing.
3. There is no evidence that after setting this array that values in it are evaluated for any purpose.
4. Even the return function does not care about it hence this appears to me that we are just using memory allocation for something that is not used in other places to verify or check with.

Reply:
I don't get why we're removing this.  If it's doing no harm then I'd prefer to keep this in.  There may be something it's used for that we haven't considered.

Reply:
Isn't the job data needed for the actual job interface in the CMS so users can see what's been going on. I think it needs to stay.

Reply:
Reverted